testing/kmouth/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+# Maintainer: Bart Ribbers <bribbers@disroot.org>
+pkgname=kmouth
+pkgver=19.04.2
+pkgrel=0
+arch="all"
+url="https://kde.org/applications/utilities/org.kde.kmouth"
+pkgdesc="Speech Synthesizer Frontend"
+license="GPL-2.0-or-later"
+makedepends="extra-cmake-modules qt5-qtbase-dev qt5-qtspeech-dev kcrash-dev kcompletion-dev kconfig-dev kconfigwidgets-dev kcoreaddons-dev kdoctools-dev ki18n-dev kio-dev kwidgetsaddons-dev kxmlgui-dev"
+checkdepends="xvfb-run"
+source="https://download.kde.org/stable/applications/$pkgver/src/$pkgname-$pkgver.tar.xz"
+subpackages="$pkgname-doc $pkgname-lang"
+
+build() {
+	cmake \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib
+	make
+}
+
+check() {
+	CTEST_OUTPUT_ON_FAILURE=TRUE ctest
```

Comment:
xvfb-run in checkdepends but not used (used implicitly ?)

Reply:
Sorry my mistake, checkdepends is not needed. Fixed it.